src/test/resources/org/folio/processing/mapping/authority/parsedRecordWithoutSourceFile.json

```diff
@@ -14,7 +14,8 @@
       "010": {
         "subfields": [
           {
-            "a": "abc   38420221 "
+            "a": "abc   38420221 ",
+            "a": "na   38420221 "
```

Comment:
I think their you try to test this:
![image](https://github.com/folio-org/data-import-processing-core/assets/5904625/4fd5dff3-219c-462d-9792-664ecac867a7)
If it's true, I suggest to just replace "abc   38420221" with "na   38420221"

Reply:
yes, this is a case where authority source file is not found, I added one more subfield - "na 38420221" which is not matching same like "abc 38420221".